subset_datasets.py

```diff
@@ -83,10 +85,19 @@
                              values='mutation', fill_value=0)
                 .astype(bool).astype(int))
 
+# Process Xena mutations
+xena_mut_df = (xena_mut_df.query("effect != 'Silent'")
+               .groupby(['#sample', 'chr', 'gene'])
+               .apply(len).reset_index()).rename(columns={0: 'mutation'})
+
+xena_mut_df = (xena_mut_df.pivot_table(index='#sample', columns='gene',
+               values='mutation', fill_value=0) .astype(bool).astype(int))
+
 # Write subsets to file
```

Comment:
`header=True` is default for `pandas.DataFrame.to_csv`. Did you mean something else?